Resources/Prototypes/Catalog/Cargo/cargo_vending.yml

```
@@ -33,7 +33,7 @@
     sprite: Objects/Specific/Service/vending_machine_restock.rsi
     state: base
   product: CrateVendingMachineRestockClothesFilled
-  cost: 7250 #DeltaV
+  cost: 8050
```

```suggestion
  cost: 8250
```
8050 is weird looking :(

---

I like it better than 8000